### PR TITLE
Lag compensation

### PR DESCRIPTION
**Pull request by klautcomputing**

### CODE REVIEW

File: apis/src/lag_tracking/lag_tracker.rs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        }
+    }
+
+    pub fn comp_est_std_err(&self) -> Option<f64> {
+        if self.moves() > 2 {
+            Some(self.comp_est_sq_err.sqrt() / ((self.moves() - 2) as f64))
+        } else {
+            None
+        }
+    }
```
Comment (nrobinaubertin) [inline apis/src/lag_tracking/lag_tracker.rs:82]: I don't think that this is used anywhere

File: apis/src/lag_tracking/lag_tracker.rs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+#[derive(Clone, Debug)]
+pub struct LagTracker {
+    pub quota_gain: f64,
+    pub quota: f64,
+    pub quota_max: f64,
+    pub lag_estimator: DecayingStats,
+    pub uncomp_stats: Stats,
+    pub lag_stats: Stats,
+    pub comp_est_sq_err: f64,
+    pub comp_est_overs: f64,
```
Comment (nrobinaubertin) [inline apis/src/lag_tracking/lag_tracker.rs:12]: I think that those two values are not really used to calculate the compensation in the end

**Review by nrobinaubertin — COMMENTED** [review]
Some leftovers from the lichess code translation but LGTM overall

**Comment by klautcomputing** [comment]
@nrobinaubertin True but I thought I might keep them around because I want to make a statistics page where you can see all the numbers which might in the future help us fine tune the lag compensation :)

**Review by IongIer — APPROVED** [review]

**Pull request merged**